### PR TITLE
wip-First look to the guide

### PR DESCRIPTION
**Pull request by giuse2596**

### CODE REVIEW

**Comment by frisso** [comment]
Giuseppe,
first, thanks for this commit!
One minor note: I would suggest not to add too many things to this document. For example, I would avoid explaining the Go syntax here, as there are many documents out there that are probably even better, and we don't want to reinvent the wheel.
I would suggest to focus on *pointers* that may help a newbie student. The problem of a new student is where does it start learning. So, pointers like "the main documentation is here", or "the best editor is here", "a short video (30-60 min) to introduce you into the topic" etc.
To me, those are the most valuable resources for a new student: a learning path to start with, considering that our student may not willing to be 100% proficient in go, but has to carry on his project in a small amount of time and hence has to get access to good documentation, but that does not require one month of full work to do the first "hello world" program.
Thanks!

**Comment by giuse2596** [comment]
Thank you for feedback, I will delete all useless sections and so I will terminate this guide.

File: content/resources/golang/golang.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+fragment = "content"
+#disabled = true
+date = "2020-12-12"
+weight = 100
+# title = "Simple guide: Golang language"
++++
+
+## Introduction
+
+Crownlabs backend is written in Golang (a.k.a. Go), an open source, procedural programming language created by Google. To simplify it can be compared with C and C++ but without some of their most tricky issues (e.g. memory management). This guide will do a general excursus on this very raising language introducing its most relevant features, giving a roadmap to follow to learn it.
```
Comment (frisso) [inline content/resources/golang/golang.md:11]:
```suggestion
Crownlabs backend is written in Golang (a.k.a. Go), an open source, procedural programming language created by Google.
In brief, it looks similar to C and C++ but without some of their most tricky issues (e.g. memory management). This guide will do a general excursus on this fast raising programming language, introducing its most relevant features and giving a roadmap for a possible learning path.
```

File: content/resources/golang/golang.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+## Useful links
+
+### Install Go
+
+The first step is to install Go and it can be done easly [here](https://golang.org/doc/install).
+
+### Ides and editor plugins
+
+To write code in Go there are some ides like [Visual Studio Code](https://code.visualstudio.com/) or [Goland](https://www.jetbrains.com/go/) and text editor plugins like [vim-go](https://github.com/fatih/vim-go).
```
Comment (frisso) [inline content/resources/golang/golang.md:21]:
```suggestion
To write code in Go, we suggest some IDEs such as [Visual Studio Code](https://code.visualstudio.com/) or [Goland](https://www.jetbrains.com/go/) and text editor plugins like [vim-go](https://github.com/fatih/vim-go).
```

File: content/resources/golang/golang.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+### Suggestions
+
+Since Go is similar to other famous programming language (e.g. C/C++) it is very simple to learn sintax and for this purpose you can use [official documentation](https://golang.org/doc/) which is very simple and clear with a lot of tutorials and examples.
+
+Very useful to the process of learning is [Tour of Go](https://tour.golang.org/list) where you can find many interactive tutorials to understand all parts of the language: you can do these little lectures in any order you like creating your personal learning path.
+
+## Meet authors
+
+If you are interested in this lenguage you can better understand it and the idea behind it watching this [video](https://youtu.be/3yghHvvZQmA) where you can listen Go creators speaking at a Google Cloud conference.
```
Comment (frisso) [inline content/resources/golang/golang.md:34]:
```suggestion
If you are interested in this language, you can better understand it and the idea behind it watching this video [(Meet the Authors – Go Language)](https://youtu.be/3yghHvvZQmA) where you can listen Go creators speaking at a Google Cloud Next 2019 conference. 
```

**Review by frisso — APPROVED** [review]

**Pull request merged**